doc/source/dicom/spm_dicom.rst

```diff
@@ -269,7 +269,7 @@ Then define the following matrices:
 .. math::
 
    R = \left(\begin{smallmatrix}1 & a & 1 & 0\\1 & b & 0 & 1\\1 & c & 0 & 0\\1 & d & 0 & 0\end{smallmatrix}\right)
-   
+
```

Comment:
Is this whitespace significant?

Reply:
No, I assume it's just the `end-of-file-fixer` hook.

Reply:
Right. I just meant we should verify that this doesn't change the appearance (or the change is acceptable) of the rendered doc.

Reply:
:sweat_smile: :sweat_smile: 
Just verified, no change in appearance.